robotbase/src/main/java/frc/robot/subsystems/AlgaeRunner.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.SparkMax;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ALGAE_RUNNER;
+import frc.robot.Constants.CAN_ID;
+
+public class AlgaeRunner extends SubsystemBase {
+    private SparkMax m_motor;
+
+    public AlgaeRunner() {
+        m_motor = new SparkMax(CAN_ID.ALGAE_RUNNER_MOTOR, MotorType.kBrushless);
+
+        m_motor.configure(
+                ALGAE_RUNNER.MOTOR_CONFIG,
+                ResetMode.kResetSafeParameters,
+                PersistMode.kPersistParameters);
+    }
+
+    public void setAxisSpeed(double percentOutput) {
```

Comment:
This method should be multiplying the percentOutput by a maximum set in the constants similar to https://github.com/frc2357/robotlib2357/blob/main/Lib2025/src/main/java/frc/robot/subsystems/Intake.java#L75 
This method will be used for any manual driver controls and we'll want to be able to limit that to a maximum.

We'll also want a `set` method that doesn't limit the percentOutput which will only be used for automatic actions performed in the code.